custom/icds_reports/utils/aggregation_helpers/tests/sql_output/growth-monitoring-forms.distributed.txt

```diff
@@ -131,12 +132,8 @@ DELETE FROM "icds_dashboard_growth_monitoring_forms" WHERE month=%(month)s AND s
                         timeend RANGE BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING
                 )
         ) ucr
-          FULL OUTER JOIN "icds_dashboard_growth_monitoring_forms" prev_month
+          FULL OUTER JOIN (SELECT * FROM "icds_dashboard_growth_monitoring_forms" WHERE month = %(previous_month)s) prev_month
```

Comment:
what's the performance of this like?

Reply:
So this is potentially the best change out of this PR. Because this is a `FULL OUTER JOIN` it was doing an sequential scan of the entire table and then the where clause was filtering out the other months via the coalesce statement commented on earlier.

I'm not sure that we can get better than this query as we do want the entire previous month of growth monitoring forms so other filters are not necessary.

Another option I briefly considered was an initial `INSERT` from either the UCR or previous month and an `INSERT ... ON CONFLICT UPDATE` from the other table. However I think that's a bigger diversion from the current setup than I've had a chance to fully think through yet